docs/authentication/enterprise-connections/saml/azure.mdx

```diff
@@ -17,39 +17,60 @@ description: Learn how to allow users to sign up and sign in to your Clerk app w
     }
   ]}
 >
-  - Use Microsoft Azure Entra ID to enable SSO via SAML for your Clerk app
+  - Create a SAML SSO connection for your customer
+  - Give your customer instructions on how to connect Microsoft Azure Entra ID to your app
```

Comment:
Up to discussion: Why not using the "organization" term instead of "customer" here to align with our product modelling